nuxeo-drive-core/src/test/java/org/nuxeo/drive/service/adapter/TestFileSystemItemAdapterService.java

```diff
@@ -436,16 +431,12 @@ public void testContribOverride() throws Exception {
 
         TransactionHelper.commitOrRollbackTransaction();
         try {
-            Framework.getRuntime().standby(Instant.now().plus(Duration.ofSeconds(30)));
-            try {
-                harness.deployContrib("org.nuxeo.drive.core.test",
-                        "OSGI-INF/test-nuxeodrive-adapter-service-contrib-override.xml");
-            } finally {
-                Framework.getRuntime().resume();
-            }
+
+            deployer.deploy("@org.nuxeo.drive.core.test:OSGI-INF/test-nuxeodrive-adapter-service-contrib-override.xml");
```

Comment:
Finally, with the new system, can't we use `@LocalDeploy` here instead? In which case keep `registerRootAndCreateSomeDocs` as a `@Before` method and not call it explicitly at the beginning of each test?